Dockerfile

```diff
@@ -231,7 +231,7 @@ RUN --mount=type=secret,uid=1000,id=private_repo_token \
 # - files owned by postgres can be overwritten in a running container
 # - new files can be added to the directories mentioned here
 RUN for pg in ${PG_VERSIONS}; do \
-        for dir in "$(/usr/lib/postgresql/${pg}/bin/pg_config --sharedir)/extension" "$(/usr/lib/postgresql/${pg}/bin/pg_config --pkglibdir)" "$(/usr/lib/postgresql/${pg}/bin/pg_config --bindir)"; do \
+        for dir in /usr/share/doc "$(/usr/lib/postgresql/${pg}/bin/pg_config --sharedir)/extension" "$(/usr/lib/postgresql/${pg}/bin/pg_config --pkglibdir)" "$(/usr/lib/postgresql/${pg}/bin/pg_config --bindir)"; do \
```

Comment:
so you want manpages inside the container?

Reply:
oh, possibly, we need /usr/share/doc to install dpkg packages?

Reply:
No, they are actually removed afterwards, in the cleanup phase.

However, these `*.deb` files that we install *include* manpages, so we have to allow them to exist.